interview-questions.md

```diff
@@ -7,61 +7,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer: An instance variable is run at the same time as the code is being typed. 
+  Your answer:
 
-  Researched answer: The main difference is its behavior with inheritance. Class variables are shared between a class and all its subclasses, while class instance variables only belong to tone specific class.
+  Researched answer:
 
 
 
 2. What is a block in Ruby?
 
-  Your answer: Ruby block is used in methods. (I honestly totally didn't know) 
+  Your answer:
 
-  Researched answer: Ruby block may appear only in the source adjacent to a method call. It is written on the same line as the method calls last parameter (the closing parentheses of the list). The code in the block is not executed at the same time it is encountered.
+  Researched answer:
 
 
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer: I think it might be something with the setter and getter or methods?
+  Your answer:
 
-  Researched answer: It returns the value of the last evaluated expressions. getter is the most commonly used.
+  Researched answer:
 
```

Comment:
Ruby automatically returns the last line of any method without expressly saying "return"